cli/src/commands/tui/app.rs

```diff
@@ -104,9 +105,10 @@ impl App {
                 builds: vec![],
                 deployments: vec![],
                 last_log_entry_timestamp: None,
-                log_entries_hash_map: HashMap::new(),
-                log_entries_ids: vec![],
                 has_log_errors: false,
+                log_entries_length: 0,
```

Comment:
I believe we don't need this, we should be able to get the len using `log_entries.len()`